composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -198,8 +198,8 @@
     end
 
     context "when the user is ignoring the latest version" do
-      let(:ignored_versions) { [">= 1.22.0.a, < 3.0"] }
-      it { is_expected.to eq(Gem::Version.new("2.8.0")) }
+      let(:ignored_versions) { [">= 1.22.0.a, < 4.0"] }
```

Comment:
This is weird... when this is at `< 3.0`, the test returns `Gem::Version.new("2.9.1")`

But when I bump it up to `< 4.0`, the test returns `Gem::Version.new("1.2.1")`

??!